detection-rules/link_microsoft_low_reputation.yml

```diff
@@ -104,19 +102,20 @@ source: |
     or any(attachments,
            .file_type in $file_types_images
            and any(file.explode(.),
-                   any(ml.nlu_classifier(.scan.ocr.raw).intents, .name == "cred_theft")
+                   any(ml.nlu_classifier(.scan.ocr.raw).intents, .name == "cred_theft" and .confidence != "low")
```

Comment:
could we be explicit instead: medium, high? i think that's usually clearer to the user